### PR TITLE
Remove Babel

### LINKED ISSUE #1062: Remove Babel
The tests are written as ES6 modules and transpiled to CommonJS using Babel. I don't think there are other features that need transpiling. It would be nice to remove Babel to simplify the developer experience.

### PR DESCRIPTION
**Pull request by nventuro**
We no longer use `import` :tada: 

Fixes #1062

### CODE REVIEW

**Comment by shrugs** [comment]
do we use jshint at all, actually?

**Comment by shrugs** [comment]
LGTM after rebase @nventuro

File: test/Bounty.test.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { ethGetBalance, ethSendTransaction } from './helpers/web3';
+const { ethGetBalance, ethSendTransaction } = require('./helpers/web3');
+
+var SecureTargetBounty = artifacts.require('SecureTargetBounty');
```
Comment (frangio) [inline test/Bounty.test.js]: Can we use `const` for these?

File: package-lock.json [hunk 2]
```diff
@@ -1246,37 +1246,6 @@
...
-              "version": "0.11.1",
-              "resolved": "https://registry.npmjs.org/regenerator-runtime/-/regenerator-runtime-0.11.1.tgz",
-              "integrity": "sha512-MguG95oij0fC3QV3URf4V2SDYGJhJnJGqvIIgdECeODCT98wSWDAJ94SSuVpYQUoTcGUIL6L4yNB7j1DFFHSBg==",
-              "dev": true
-            }
-          }
-        }
-      }
-    },
     "babel-preset-env": {
```
Comment (frangio) [inline package-lock.json]: This should've probably been removed, right? Re-run `npm install`?

File: test/Bounty.test.js (hunk 1 above)
Comment (nventuro) [reply test/Bounty.test.js]: Yes, but we're only removing Babel in this PR, that's coming on a separate one to keep everything tidy.

File: package-lock.json (hunk 2 above)
Comment (shrugs) [reply package-lock.json]: I looked into that as well, but they're dev dependencies of solidity-coverage, so they're in our package-lock

File: test/Bounty.test.js (hunk 1 above)
Comment (nventuro) [reply test/Bounty.test.js]: Created #1091.

File: package-lock.json (hunk 2 above)
Comment (frangio) [reply package-lock.json]: Ah, got it. :slightly_smiling_face:

**Comment by sweatyc** [comment]
that would be nice, since my not a Node.js expert and currently scratching my head trying to use the helper scripts for test but `import` and `export` don't work...

**Comment by nventuro** [comment]
@sweatyc what do you mean? All helpers are imported using `require` now that this PR has been merged, a testing guide with information on how to use them, example, etc., is on the way (#1077).

**Pull request merged**